docs/operators/jdbcPOJOInsertOutputOperator.md

```diff
@@ -0,0 +1,171 @@
+JDBC Transactional POJO Output Operator
+==============
+
+## Operator Objective
+This operator receives an input stream of POJOs and inserts them as rows in a database table in a fault-tolerant way.
+
+## Overview
+The main features of this operator are persisting data to the database table and fault tolerance. This operator creates a transaction at the start of each window, executes batches of sql updates, and closes the transaction at the end of the window. Each tuple corresponds to an SQL update statement. The operator groups the updates in a batch and submits them with one call to the database. Batch processing improves performance considerably. The size of a batch is configured by `batchSize` property. The tuples in a window are stored in a check-pointed collection which is cleared in the `endWindow()`. The operator writes a tuple exactly once in the database.
```

Comment:
Mention the specific name of the class you are discussing, e.g.

The main features of this operator(`AbstractPassThruTransactionableStoreOutputOperator`) are ....